shunting-yard-replicator/src/main/java/com/hotels/shunting/yard/replicator/exec/messaging/MessageReaderAdapter.java

```diff
@@ -74,6 +76,10 @@ private MetaStoreEvent map(ListenerEvent listenerEvent) {
       break;
     case ALTER_PARTITION:
       AlterPartitionEvent alterPartition = (AlterPartitionEvent) listenerEvent;
+
+      if (alterPartition.getPartitionLocation() == alterPartition.getOldPartitionLocation()) {
```

Comment:
Any particular reason for using `==` over `.equals()` - I usually prefer the latter as it's more type safe (see https://stackoverflow.com/questions/513832/how-do-i-compare-strings-in-java). This probably works most of the time but there's no guarantee that the strings in question will interned and thus have equal object references.

Reply:
Totally missed this. Fixed now.